### PR TITLE
Update: the usage instructions for parameter <ConsiderFixturesAsSpecialTests>

### PR DESCRIPTION
**Pull request by wenbo97**
### Describe your changes here.
Hi @Evangelink @IEvangelist

Add `ConsiderFixturesAsSpecialTests` usage based on discussion: https://github.com/microsoft/testfx/issues/3785

### Fixes #Issue_Number (if available)
No issue number.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/testing/unit-testing-mstest-configure.md](https://github.com/dotnet/docs/blob/44ac49da33574dc5a1b70bbeeb65ac6123525987/docs/core/testing/unit-testing-mstest-configure.md) | [docs/core/testing/unit-testing-mstest-configure](https://review.learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-mstest-configure?branch=pr-en-us-42567) |

### CODE REVIEW

File: docs/core/testing/unit-testing-mstest-configure.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ The following runsettings entries let you configure how MSTest behaves.
 |**TestTimeout**|0|Gets specified global test case timeout.|
 |**TreatClassAndAssemblyCleanupWarningsAsErrors**|false|To see your failures in class cleanups as errors, set this value to **true**.|
 |**TreatDiscoveryWarningsAsErrors**|false|To report test discovery warnings as errors, set this value to **true**.|
+|**ConsiderFixturesAsSpecialTests**|false|To display `AssemblyInitialize`, `AssemblyCleanup`, `ClassInitialize`, `ClassCleanup` as individual entries in Visual Studio `Test Explorer` and `.trx` log, set this value to **true**|
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-mstest-configure.md:46]:
```suggestion
|**ConsiderFixturesAsSpecialTests**|false|To display `AssemblyInitialize`, `AssemblyCleanup`, `ClassInitialize`, `ClassCleanup` as individual entries in Visual Studio and Visual Studio Code `Test Explorer` and `.trx` log, set this value to **true**|
```

**Review by Evangelink — APPROVED** [review]
Thank you @Vincezhaoc

File: docs/core/testing/unit-testing-mstest-configure.md (hunk 1 above)
Comment (Vincezhaoc) [reply docs/core/testing/unit-testing-mstest-configure.md]: Hi @Evangelink,

Fixed.

**Review by Evangelink — APPROVED** [review]
@IEvangelist Could you please review?

File: docs/core/testing/unit-testing-mstest-configure.md [hunk 2]
```diff
@@ -43,6 +43,7 @@ The following runsettings entries let you configure how MSTest behaves.
 |**TestTimeout**|0|Gets specified global test case timeout.|
 |**TreatClassAndAssemblyCleanupWarningsAsErrors**|false|To see your failures in class cleanups as errors, set this value to **true**.|
 |**TreatDiscoveryWarningsAsErrors**|false|To report test discovery warnings as errors, set this value to **true**.|
+|**ConsiderFixturesAsSpecialTests**|false|To display `AssemblyInitialize`, `AssemblyCleanup`, `ClassInitialize`, `ClassCleanup` as individual entries in Visual Studio and Visual Studio Code `Test Explorer` and `.trx` log, set this value to **true**|
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-configure.md:46]:
```suggestion
|**ConsiderFixturesAsSpecialTests**|false|To display `AssemblyInitialize`, `AssemblyCleanup`, `ClassInitialize`, `ClassCleanup` as individual entries in Visual Studio and Visual Studio Code `Test Explorer` and _.trx_ log, set this value to **true**|
```

**Review by IEvangelist — APPROVED** [review]

**Comment by Vincezhaoc** [comment]
Thank you @IEvangelist @Evangelink

**Pull request merged**